plugin.xml

```diff
@@ -29,7 +29,7 @@
 		<hook type="before_plugin_install" src="hooks/outsystems/handle_google_services.js" />
 		<hook type="after_plugin_install" src="hooks/outsystems/google_services_cleanup.js" />
 
-		<dependency id="cordova-support-google-services" version="1.2.1" />
+		<dependency id="cordova-support-google-services" version="1.3.1" />
```

Comment:
We should open a PR for the upstream or at least challenge Pushwoosh to upgrade that version. The same for the locked version in firebase dependencies, as we're locking it, I can see in the future we ignoring that and move for the latest versions.